dcicutils/deployment_utils.py

```diff
@@ -265,19 +288,21 @@ def create_indexer_configuration_template(cls, env_name, size=None):
 
         # upload the template
         if is_cgap_env(env_name):
-            return eb_client.create_configuration_template(
+            eb_client.create_configuration_template(
                 ApplicationName=cls.EB_APPLICATION,
                 TemplateName=CGAP_ENV_INDEXER,
```

Comment:
Feels like we need a function indexer_env_for_environment(env) that returns either cgap_env_indexer or ff_env_indexer so that you can share these two bodies of code.